src/server.py

```diff
@@ -153,5 +158,22 @@ def docs_fonts_path(path):
     return send_from_directory('docs/static/libs/bootstrap-3_3_7-dist/fonts', path)
 
 
+@app.route('/schema/<string:class_or_prop>')
+def dereference_class_or_prop(class_or_prop):
+    accept_type = request.headers.get('Accept')
+    if 'text/turtle' in accept_type:
+        uri = "http://data.rdlabs.beeldengeluid.nl/schema/%s" % class_or_prop
+        return sim.get_resource(class_or_prop=uri)
+    else:
+        # return pages are either class or prop pages generated by ontospy in docs/ folder
+        doc_title = 'class-nisv%s.html' % class_or_prop.lower()
+        print(request.path)
+        if os.path.exists(os.path.join(Path(app.static_folder).parent, 'docs/%s' % doc_title)):
+            return redirect(os.path.join('/docs', doc_title))
```

Comment:
@wmelder het gebruik van een redirect is hier niet gepast, want nu kan je b.v. niet makkelijk met `curl` of een andere command line tool de contents achter dit pad ophalen, omdat redirects vaak niet ondersteund worden daar.